### PR TITLE
Metrics webapp/ Stocks Watcher

### PR DESCRIPTION
**Pull request by MaiRizk**
I have used the stocks data API to build this project. 
All the project requirements have been met in this project and it's working fine on the browser. 
All tests are passing. 
The website is deployed on Netlify.

Here's the video link: https://www.loom.com/share/dcd5142de4c2462b9eab7c94f02928fe

### CODE REVIEW

File: README.md [hunk 1]
```diff
@@ -1,70 +1,76 @@
...
+# My Stocks Watcher App
 
-This project was bootstrapped with [Create React App](https://github.com/facebook/create-react-app).
+## About
 
-## Available Scripts
+This mobile web application was developed as the Capstone Project for Microverse React-Redux Curriculum Module. It was built upon [Financial Modeling Prep API](https://financialmodelingprep.com/developer/docs/) and the original design idea was created by [Nelson Sakwa on Behance](https://www.behance.net/sakwadesignstudio). My task was to check a list of metrics (data stock, in my case) making use of React and Redux. The application allows users to list the Stocks, ETFs, and Funds traded in AMEX, NYSE, and Nasdaq Stock Exchanges and filter their search.
 
-In the project directory, you can run:
+Below, I present some screenshots of the app.
```
Comment (arslanbisharat) [inline README.md:7]: - [ ] Please add a Video link here as requested in guidelines.

File: src/utils/parseData.js [hunk 2]
```diff
@@ -0,0 +1,26 @@
+/* eslint-disable no-param-reassign */
```
Comment (arslanbisharat) [inline src/utils/parseData.js]: - [ ] Let's not disable linter rules. These are here help us write best code.

**Review by arslanbisharat — CHANGES_REQUESTED** [review]
# Changes Requested, 

Hi there, 

Good job so far!
There are some issues that you still need to work on to go to the next project but you are almost there!

 ## Required Changes ♻️
_Check the comments under the review._

## Optional suggestions
_Every comment with the [OPTIONAL] prefix is not crucial enough to stop the approval of this PR. However, I strongly recommend you to take them into account as they can make your code better._

You can also consider:
- [ ]  Let's make sure you are following correct Gitflow. The Pull Request should be from feature to development.


Cheers and Happy coding!👏👏👏

Feel free to leave any questions or comments in the PR thread if something is not 100% clear.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have limited number of code reviews per project. If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._
```

**Review by mikenath223 — APPROVED** [review]
Hello there  

Your project is complete!  There is nothing else to say other than... it's time to merge it 🛳 
Congratulations! 🎉 

Happy coding. Cheers! 🚀🚀

If you feel something isn't clear enough you can leave a question or comment in the PR thread.

------
_As described in the [Code reviews limits policy](https://microverse.zendesk.com/hc/en-us/articles/1500004088561) you have a limited number of reviews per project (check the exact number in your Dashboard). If you think that the code review was not fair, you can request a second opinion using [this form](https://airtable.com/shrQAqnBwek5a0O0s)._

**Pull request merged**